01-introduction.Rmd

```diff
@@ -112,30 +106,43 @@ leaflet() %>%
   addAwesomeMarkers(lng = c(-3, 23), lat = c(52, 53), popup = popup)
 ```
 
-It would be difficult to produce Figure \@ref(fig:interactive) with only four lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
-The use of R code therefore enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
-<!-- Other reasons for using R for geocompuation include: -->
+It would have been difficult to produce Figure \@ref(fig:interactive) using R a few years ago, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)).
+This illustrates R's flexibility and how, thanks to developments such as **knitr** and **leaflet**, it can be used as an interface to other software, a theme that will recur throughout this book.
+The use of R code, therefore, enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
 
-### Python, Java, C++
+## Other software for geocomputation
```

Comment:
I would just call it Software for Geocomputation - it already mentions R, so I think the the worth 'Other' is superfluous and unnecessarily verbose.